votor/src/consensus_pool/stats.rs

```diff
@@ -46,6 +45,40 @@ impl CertificateStats {
     }
 }
 
+#[derive(Default)]
+struct VoteStats {
+    notarize: u64,
+    finalize: u64,
+    skip: u64,
+    notarize_fallback: u64,
+    skip_fallback: u64,
+}
```

Comment:
Err, how did this sneak in? Can this go into another PR because your title didn't claim to do this.

Also I think this is in another PR?